presto-testing/pom.xml

```diff
@@ -47,6 +47,11 @@
             <artifactId>presto-tpch</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.prestosql</groupId>
+            <artifactId>presto-plugin-toolkit</artifactId>
```

Comment:
is it avoidable?

i am concerned we could want to have dep the other way -- use `testing` goodies in toolkit's tests